AngularTask-3/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+import { GreetPipe } from './pipes/greet.pipe';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
+      ],
+      declarations: [
+        AppComponent,
+        GreetPipe
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'testing'`, () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app.title).toEqual('testing');
+  });
+
+ 
```

Comment:
Please try adding basic test for component testing aspect
- Accessing the rendered DOM and checking text content
- Simulating user input like clicks and form field input
- Testing Component Input and Outputs